packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet.tsx

```diff
@@ -68,12 +69,38 @@ export class AtomicSmartSnippet implements InitializableComponent {
    * When the answer is partly hidden, how much of its height (in pixels) should be visible.
    */
   @Prop({reflect: true}) collapsedHeight = 180;
+  /**
+   * Sets the style of the snippet.
+   *
+   * Example:
+   * ```ts
+   * smartSnippet.snippetStyle = `
+   *   b {
+   *     color: blue;
+   *   }
+   * `;
+   * ```
+   *
+   * Alternatively, you can style the snippet by inserting a template element like this:
```

Comment:
Although using a `template` with a `style` element is pretty much the "primary way" of doing it, I think it's easier to document like this.

Reply:
Why do you find this easier ? 🤔  

My first instinct would have been to document the recommended/primary way first, with explicit keywords that explains/reassure readers that "this is the recommended way".

And not leave them to figure it out on their own.